src/main/java/org/cs1331/gitsubmitter/Application.java

```diff
@@ -111,22 +54,34 @@ public static void main(String... args) throws Exception {
                 System.out.println("...done");
             }
 
-            String repoName = line.getOptionValue(REPO_OPT);
+            String repoName = line.getOptionValue(REPO_NAME_OPT);
+
+            // Currently Not Used
+            String username = null;
+            if (line.hasOption(USERNAME_OPT)) {
+                username = line.getOptionValue(USERNAME_OPT);
+            }
+
+            // Currently Not Used
+            char[] password = null;
+            if (line.hasOption(PASSWORD_OPT)) {
+                password = line.getOptionValue(PASSWORD_OPT).toCharArray();
```

Comment:
At this point, there's already a `String` in memory, so the `char[]` trick doesn't do much in terms of security.